### ISSUE TITLE
How to train and validate my own dataset?

### ISSUE THREAD

**Issue by EEWenbinWu** [HRNet/Lite-HRNet#16]
Thanks for your great works. It helps me a lot.
But your project is based on mmcv, how can I get my own dataloader?

**Comment by yu-changqian** [comment]
Hello, you could follow this [tutorial](https://github.com/open-mmlab/mmpose/blob/master/docs/tutorials/2_new_dataset.md) to add your own dataset. The custom data pipeline maybe refer to this [tutorial](https://github.com/open-mmlab/mmpose/blob/master/docs/tutorials/3_data_pipeline.md).

**Comment by EEWenbinWu** [comment]
thx，and I also have question about why you set the batchnorm layers to eval mode.
`def train(self, mode=True):
        """Convert the model into training mode."""
        super().train(mode)
        if mode and self.norm_eval:
            for m in self.modules():
                if isinstance(m, _BatchNorm):
                    m.eval()
© 2021 GitHub, Inc.`
